climpred/comparisons.py

```diff
@@ -209,8 +226,9 @@ def _m2r(ds, reference):
     has_dims(ds, 'member', 'decadal prediction ensemble')
     has_min_len(ds['member'], 1, 'decadal prediction ensemble member')
     forecast = ds
-    reference = reference.expand_dims('member')
-    nMember = forecast.member.size
-    reference = reference.isel(member=[0] * nMember)
-    reference['member'] = forecast['member']
+    if stack:
```

Comment:
What happens if this is false? There's no else statement here.

Reply:
deliberately nothing.